common/app/model/trails.scala

```diff
@@ -24,24 +24,22 @@ object Trail {
       .orElse(elements.videos.headOption.map(_.images))
       .orElse(elements.thumbnail.map(_.images))
 
-    // Try to take the master 5:3 image. At render-time, the image resizing service will size the image according to card width.
-    // Filtering the list images here means that facia-press does not need to slim down the Trail object.
+    // Try to take the master 5:3 image, the largest and the smallest 5:3 image. At render-time, the image resizing service
+    // will size the largest image according to card width.  If the resizing service is switched off, we use the smallest.
```

Comment:
Thanks. I’m a master 👮 here. And [there](https://trello.com/c/cWSiXgfV/87-all-imagery-to-properly-use-master-assets-and-resizer-responsively).

If resizer -> **always** master.